### ISSUE TITLE
Http basic auth

### ISSUE THREAD

**Issue by nodesocket** [Mashape/apiembed#6]
Is prompting for http basic auth possible? How would that function?

**Comment by ahmadnassri** [comment]
@nodesocket been contemplating on that, were you thinking in the GUI or in the code output (library methods, CLI arguments)

I'm thinking of the latter, extending the HAR format to allow for authentication specific properties, allowing library output code to use methods / handlers specific to setting authentication headers (rather than directly setting the `Authorization` header value as it does now)

**Comment by nodesocket** [comment]
For us, it would be awesome to prompt for username, password right in code output and that would simply just inject those values into each language.

Does that make sense?

**Comment by ahmadnassri** [comment]
makes sense ... will have to think about that more, keeping this ticket open for now.

**Comment by bdentino** [comment]
has this been given any more consideration?
